org.adempiere.base/src/org/compiere/acct/Doc_Inventory.java

```diff
@@ -245,6 +245,11 @@ else if (MAcctSchema.COSTINGLEVEL_BatchLot.equals(costingLevel))
 			}
 			else 
 			{
+				//if product type expense and stocked, then no needs to do posting

```

Comment:
Posting an Internal Use Inventory is throwing this error:
15:42:49.073===========> Doc_Inventory.post:  [371]
java.lang.NullPointerException: Cannot invoke "org.compiere.model.MProduct.getProductType()" because "product" is null
	at org.compiere.acct.Doc_Inventory.createFacts(Doc_Inventory.java:249)
	at org.compiere.acct.Doc.postLogic(Doc.java:741)
	at org.compiere.acct.Doc.post(Doc.java:625)